lib/bolt/cli.rb

```diff
@@ -181,9 +181,13 @@ def create_option_parser(results)
                 "Output format to use: human or json") do |format|
           results[:format] = format
         end
-        opts.on('-k', '--insecure',
-                "Whether to connect insecurely ") do |insecure|
-          results[:insecure] = insecure
+        opts.on('--[no-]host-key-check',
```

Comment:
`--nodes` help still mentions `--insecure`. We should make it clear that we use `host-key-check` and `ssl` by default.

Reply:
The README also mentions `insecure`, should be updated to reflect new options.

Reply:
Fixed